sel4-ftrace-merge/include/ftrace.h

```diff
@@ -0,0 +1,213 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
Not sure if this file is free from indentation issues either.

Reply:
here again. I already fixed all thees  indentation issues, but submitted them on different repo. Sorry.